py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -202,7 +205,9 @@ def check_response(self, response: dict[str, Any]) -> None:
             screen = value["screen"]
 
         stacktrace = None
-        st_value = value.get("stackTrace") or value.get("stacktrace")
+        st_value = None
+        if isinstance(value, dict):
+            st_value = value.get("stackTrace") or value.get("stacktrace")
```

Comment:
This instance check is not required since we are taking care of it below. It won't raise any mypy type errors.